tf/mod_source_parse_test.go

```diff
@@ -129,6 +218,27 @@ func TestParseGitSources(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:   "git::https source with subdir",
+			source: "git::https://example.com/vpc.git//subdir",
+			want: want{
+				parsed: tf.Source{
+					URL:    "https://example.com/vpc.git",
+					Path:   "example.com/vpc",
+					Subdir: "/subdir",
+				},
+			},
+		},
+		{
+			name:   "git::https source with empty subdir",
+			source: "git::https://example.com/vpc.git//",
+			want: want{
+				parsed: tf.Source{
+					URL:  "https://example.com/vpc.git",
+					Path: "example.com/vpc",
+				},
+			},
+		},
```

Comment:
```suggestion
		},
		{
			name:   "git::file:// source with subdir",
			source: "git::file:///tmp/test/repo//subdir",
			want: want{
				parsed: tf.Source{
					URL:    "file:///tmp/test/repo",
					Path:   "/tmp/test/repo",
					Subdir: "/subdir",
				},
			},
		},
```

maybe adding a test for local files as well?

Reply:
hmm at least on Terraform context this would be quite odd :thinking: , unless on testing scenarios (like ours). Will add it nonetheless, doesn't harm :+1: